### PR TITLE
add version tip

### PR DESCRIPTION
**Pull request by liuboaibc**
Signed-off-by: liuboaibc <kukudehero@gmail.com>

### CODE REVIEW

File: i18n/en.yaml [hunk 1]
```diff
@@ -117,4 +117,8 @@
 - id: Join Slack
   translation: Join Slack
 - id: Join Wechat
-  translation: Scan the QR code
\ No newline at end of file
+  translation: Scan the QR code
+- id: Version Tip
+  translation: You are viewing documentation for KubeSphere version:{{ .version }}
+- id: Version Description
+  translation: Kubernetes {{ .version }} documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see <a href='https://kubesphere.io/docs' target='_blank'>the latest version</a>.
```
Comment (FeynmanZhou) [inline i18n/en.yaml:124]: Kubernetes => KubeSphere

Comment (liuboaibc) [reply i18n/en.yaml]: done

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**